soc/arm/nordic_nrf/nrf51/soc.c

```diff
@@ -18,6 +18,7 @@
 #include <hal/nrf_power.h>
 #include <soc/nrfx_coredep.h>
 #include <logging/log.h>
+#include <../soc/arm/nordic_nrf/nrf_reset_reason.h>
```

Comment:
This shouldn't need to be relative; `soc/arm` should be found in the standard include path.